src/fight.cpp

```diff
@@ -1933,7 +1933,7 @@ int weapon_special(ObjData *wpn, CharData *ch) {
 }
 
 void hit(CharData *ch, CharData *victim, int type) {
-    int victim_ac, calc_thaco, dam, diceroll, weapon_position, hidden;
+    int victim_ac, calc_thaco, dam, diceroll, weapon_position, hidden, hit;
```

Comment:
if you're going to assign 'true' and 'false' to `hit` I would prefer you make `hit` a bool.  (`bool` didn't exist in old c code, which is why they used ints, but they're expensive and don't make sense here.)